src/mon/OSDMonitor.h

```diff
@@ -107,11 +107,11 @@ class LastEpochClean {
     std::vector<epoch_t> epoch_by_pg;
     ps_t next_missing = 0;
     epoch_t floor = std::numeric_limits<epoch_t>::max();
-    void report(ps_t pg, epoch_t last_epoch_clean);
+    void report(ps_t pg, epoch_t last_epoch_clean, const size_t pg_num);
```

Comment:
nit, 

1. might want to wrap the line at char 80
2. use `unsigned` for the type of `pg_num`
3. put `pg_num` as the first parameter, as it's a property of the pool actually. so, the ordering of the parameter makes more sense, like "pool, pg identity, pg's property".

Reply:
OK all clear, doing.